bugbug/repository.py

```diff
@@ -41,31 +60,85 @@ def __init__(
         desc,
         date,
         pushdate,
-        bug,
+        bug_id,
         backedoutby,
         author_email,
-        files,
-        file_copies,
         reviewers,
     ):
         self.node = node
         self.author = author
         self.desc = desc
         self.date = date
         self.pushdate = pushdate
-        self.bug = bug
+        self.bug_id = bug_id
         self.backedoutby = backedoutby
+        self.ever_backedout = backedoutby != ""
         self.author_email = author_email
-        self.files = files
-        self.file_copies = file_copies
+        self.files = []
+        self.file_copies = {}
         self.reviewers = reviewers
+        self.added = 0
```

Comment:
Can we avoid the default values? So we break loudly if we don't set them anymore.